source/js/components/join/join.jsx

```diff
@@ -67,7 +66,7 @@ export default class JoinUs extends React.Component {
         <div className="col">
           <div className="row">
             <div className="col-12 join-content">
-              <div className="my-3 join-page-title">
+              <div className="mb-5 join-page-title">
```

Comment:
I think there's a way to make this `my-5` and remove the `.my-5` from `sign-up.pug`. Can you try that out?